android-sdk/src/main/java/com/mobilecoin/lib/MobileCoinClient.java

```diff
@@ -618,12 +618,70 @@ public Receipt.Status getReceiptStatus(@NonNull Receipt receipt)
     @Override
     @NonNull
     public Transaction.Status getTransactionStatus(@NonNull Transaction transaction)
-            throws InvalidFogResponse, AttestationException,
-            NetworkException, FogSyncException {
+            throws NetworkException, AttestationException, FogSyncException, InvalidFogResponse {
         Logger.i(TAG, "GetTransactionStatus call");
         return getAccountSnapshot().getTransactionStatus(transaction);
     }
 
+    /**
+     * Returns transaction status without waiting for all of the Fog services to sync up.
+     * Caution: since this transaction check does not wait for all services to sync up the balance
+     * may need a bit of time to catch up to a block where the transaction was posted.
+     * To wait for all services to be in sync use regular {@link #getTransactionStatus}
+     *
+     * <pre>
+     * Transaction.Status status = mobileCoinClient.getTransactionStatusQuick(transaction);
+     * switch (status) {
+     * case Transaction.Status.ACCEPTED:
+     *     postedBlockIndex = status.getBlockIndex;
+     *         ...
+     *     }
+     * ...
+     * balance = mobileCoinClient.getBalance();
+     * balanceBlockIndex = balance.getBlockIndex();
+     *
+     * if (postedBlockIndex.compare(balanceBlockIndex) < 0) {
+     *     // balance hasn't caught up yet
+     * }
+     * </pre>
+     */
+    @NonNull
+    public Transaction.Status getTransactionStatusQuick(@NonNull Transaction transaction)
+            throws NetworkException {
+        Logger.i(TAG, "GetTransactionStatus call");
```

Comment:
```suggestion
        Logger.i(TAG, "GetTransactionStatusQuick call");
```
Should update name here to prevent confusion